tests/kuttl/tests/basic/ns-records-configmap/02-assert.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: designate-ns-records-params
+data:
+  ns_records: |
```

Comment:
I'm not sure you need the pipe here.

Reply:
I tried it without the pipe and it fails, so I will leave it there.